docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-managed-con.adoc

```diff
@@ -0,0 +1,19 @@
+[id='environment-managed-con']
+= Managed environment
+You can deploy an environment that includes several different pods running {KIE_SERVER}; no processes are initially loaded on the servers. The database servers are, by default, also run in pods. Each {KIE_SERVER} pod can be separately scaled as necessary.
+
+A pod with {CENTRAL} Monitoring and pod with Smart Router are also deployed. You can optionally use {CENTRAL} Monitoring to deploy, load and unload processes on your {KIE_SERVERS} and to view monitoring information. Otherwise, you can deploy, load and inluad processes using API calls to the {KIE_SERVERS}.
+
+Smart Router is a single endpoint that can receive calls to any of your processes and route them automatically to the server that actually runs the the process.
+
+image::openshift-pam-managed.png[Architecture diagram of a managed infrastructure]
+
+You must provide a Maven repository with the processes (KJAR files) that you want to deploy on the servers. Your integration process must ensure that the required versions of the processes are uploaded to the Maven repository. You can use the Maven repository of a development {CENTRAL} for this purpose.
```

Comment:
(KJAR files) -> KIE containers (I think that's a better and more formal way to refer to it)

Reply:
@bmozaffa I can't agree here. What is in the Maven repo is what people build in Business Central or an IDE, and that's a KJAR file. They don't build a KIE server container, or at least, for all I now, they don't think of it that way. 

Instead the KIE server loads a KJAR file, thus creating a KIE server contrainer, right?

Reply:
I would just make sure we're consistent. Whatever terminology we show on the other product documentation pages, and whatever we call it in the business central console.